kotlin/custom-dictionary-style/src/main/java/com/esri/arcgisruntime/sample/customdictionarystyle/MainActivity.kt

```diff
@@ -6,58 +6,83 @@ import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
 import com.esri.arcgisruntime.data.ServiceFeatureTable
 import com.esri.arcgisruntime.layers.FeatureLayer
 import com.esri.arcgisruntime.mapping.ArcGISMap
-import com.esri.arcgisruntime.mapping.Basemap
 import com.esri.arcgisruntime.mapping.BasemapStyle
 import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.portal.Portal
+import com.esri.arcgisruntime.portal.PortalItem
 import com.esri.arcgisruntime.symbology.DictionaryRenderer
 import com.esri.arcgisruntime.symbology.DictionarySymbolStyle
 import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.style_controls_layout.*
 
 class MainActivity : AppCompatActivity() {
 
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_main)
-
-    // authentication with an API key or named user is required to access basemaps and other 
-    // location services
-    ArcGISRuntimeEnvironment.setApiKey(BuildConfig.API_KEY)
-
-    // create a feature layer from a service feature table
-    val featureTable = ServiceFeatureTable(getString(R.string.restaurants_url))
-    val featureLayer = FeatureLayer(featureTable).apply {
-      // use a custom style to create a dictionary renderer and set it to the feature layer renderer
-      renderer = DictionaryRenderer(
-        DictionarySymbolStyle.createFromFile(
-          getExternalFilesDir(null)?.path + getString(R.string.restaurant_stylx_path)
-        )
-      )
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        // authentication with an API key or named user is required to access basemaps and other
+        // location services
+        ArcGISRuntimeEnvironment.setApiKey(BuildConfig.API_KEY)
+
+        // create a feature layer from a service feature table
+        val featureTable =
+            ServiceFeatureTable("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer/0")
+        val featureLayer = FeatureLayer(featureTable)
+
+        // create a new map with a streets basemap and set it to the map view
+        mapView.map = ArcGISMap(BasemapStyle.ARCGIS_STREETS).apply {
```

Comment:
```suggestion
        mapView.map = ArcGISMap(BasemapStyle. ARCGIS_TOPOGRAPHIC).apply {
```
Also the screenshot needs to be replaced.